docfx_yaml/extension.py

```diff
@@ -354,7 +358,11 @@ def build_finished(app, exception):
                         obj['summary'] = obj['syntax'].pop('summary')
                     # Raise up seealso
                     if 'seealso' in obj['syntax'] and obj['syntax']['seealso']:
-                        obj['seealsoContent'] = obj['syntax'].pop('seealso')
+                        seealsoContent = obj['syntax'].pop('seealso')
+                        if seealsoContent.startswith('See also: '):
+                            seealsoContent = seealsoContent[10:]  # 10 is length of 'See also: '
+                        obj['seealsoContent'] = seealsoContent
+
```

Comment:
This could also be done with a similar approach to removing the example node, I believe: https://github.com/ericholscher/sphinx-docfx-yaml/pull/16/files#diff-4d04434ff29400403a882f2d116809feR213 -- this works for now, but we should really be removing it before it makes its way into the data structure.